pkg/interface/src/views/apps/launch/components/tiles/clock.js

```diff
@@ -63,13 +63,21 @@ const circle = (ctx, x, y, r, from, to, fill) => {
   ctx.fill();
 };
 
+const circleClip = (ctx, x, y, r, from, to, fill) => {
+  ctx.globalCompositeOperation = 'xor';
+  circle(ctx, x, y, r, from, to, fill);
+  ctx.globalCompositeOperation = 'source-over';
+};
+
```

Comment:
Produces this effect in light mode with a dark background

<img width="152" alt="image" src="https://user-images.githubusercontent.com/20846414/95253153-fb508800-07eb-11eb-8d01-b6c6b08d3de9.png">

Reply:
That doesn't look good

Reply:
This is due to making the hole transparent. It just responds to light/dark mode, nothing else. Should I just fill in the hole? I don't see a good way to detect background color, especially with images.

Reply:
This is what I usually use `mix-blend-mode` for but it needs a background so you'd need to like *go up the DOM* and inherit background colors or set it according to a prospective contrast ratio? Idk man elaborate shit

Reply:
If you want to just call it for now you can plug the hole, it's not ideal but better than the messy case here

Reply:
Ok, I think I will. In the future I'm going to rewrite this in SVG so we can use mix-blend-mode (and to draw the moon phase), but I'll just plug the hole. Don't tell Ed, he'll be heartbroken